src/handler/make_handler.jl

```diff
@@ -59,6 +128,11 @@ function make_handler(app::DashApp, registry::ResourcesRegistry; check_layout =
     add_route!(process_index, router, "$prefix")
 
     handler = state_handler(router, state)
+    if get_devsetting(app, :prune_errors)
+        handler = exception_handling_handler(debug_exception_handling, handler)
+    else
+        handler = exception_handling_handler(exception_handling, handler)
+    end
```

Comment:
On the Python side, `prune_errors` just refers to whether the Flask and Dash parts of the stack trace should be pruned away, leaving only the user's app code in the stack trace. If I'm understanding this correctly, `debug_exception_handling` sends the error in the response (always pruned to just the user's app code, per `while !isnothing(it) && it[1].func != :process_callback_call` I guess?) whereas `exception_handling` returns a blank 500 response, is that right?

Ideally we'd have three cases:
- `debug=false` - blank 500 response on error, for security purposes. Error is only shown in the server log.
- `debug=true, prune_errors=false` - show the complete stack trace in the browser
- `debug=true, prune_errors=true` - show only the user-portion of the stack trace in the browser

Reply:
@alexcjohnson 
I agree. But still, it seems convenient to do prune in the console log, too, regardless of debug, only by prune_errors. The part of the trace that falls on the internal architecture of Dash when an error occurs in the callback is quite large and most often it is not interesting to the developer of the dashboard.

Reply:
Good point - `prune_errors` can also determine what shows up in the server log. My concern is it looked like in your code `prune_errors` was being used to determine whether the error goes to the server log or gets returned in the response, but THAT part should be controlled by `debug`

Reply:
(as discussed elsewhere, @waralex points out that `ui` is even better than `debug` for whether to send tracebacks to the front end)

Reply:
@alexcjohnson done at 76af75b